src/transformers/models/whisper/modeling_tf_whisper.py

```diff
@@ -1288,7 +1284,7 @@ def call(
         >>> inputs = processor(ds[0]["audio"]["array"], return_tensors="tf")
         >>> input_features = inputs.input_features
 
-        >>> generated_ids = model.generate(inputs=input_features)
+        >>> generated_ids = model.generate(input_ids=input_features)
```

Comment:
Is this change necessary? I believe it works with `inputs` (and should work if not).

Reply:
I believe so. I'm unable to pass in `inputs` to the tensorflow model generate function and this matches with what I see in the code. 

[PT generate function](https://github.com/huggingface/transformers/blob/df8faba4db89897e6a1da4b28a17c01aa24e3b7e/src/transformers/generation_utils.py#L917) takes `inputs`

[TF generate function](https://github.com/huggingface/transformers/blob/df8faba4db89897e6a1da4b28a17c01aa24e3b7e/src/transformers/generation_tf_utils.py#L380) takes `input_ids`. `inputs` would be passed in as `model_kwargs`

I agree that the inputs should be standardised if they're not already. I think that might be a bit of work to make sure it's updated safely if it needs to be cc @gante

Reply:
(added to the TODO list -- I was aware of the problem but hadn't placed it there :) )